frontend/integration-tests/views/sidenav.view.ts

```diff
@@ -1,10 +1,16 @@
-import { $$, by, browser, element, ExpectedConditions as until } from 'protractor';
+/* eslint-disable no-undef, no-unused-vars */
```

Comment:
This shouldn't be necessary any longer for TypeScript files. Can you try removing and seeing if you still have errors?